ansible/templates/topo/t0-64.j2

```diff
@@ -61,7 +61,7 @@
             <RouteMapOut i:nil="true"/>
             <Vrf i:nil="true"/>
             <a:Name>BGPVac</a:Name>
-            <a:PeersRange>192.168.0.0/27</a:PeersRange>
+            <a:PeersRange>192.168.0.0/26</a:PeersRange>
```

Comment:
/21 is better. It allows to have 2048 hosts. 2000 is target for the fast reboot test